docker-compose.yml

```diff
@@ -157,7 +157,7 @@ services:
         condition: service_healthy
 
   loadPgData:
-    image: node:12
+    image: node:12-bullseye
```

Comment:
We should do the same in the docker-compose-elastic-cloud.yml probably, though I am not sure when/where that is used.